hs_file_types/models/base.py

```diff
@@ -1549,6 +1549,9 @@ def __init__(self, aggr_cls, user, resource, file_id=None, folder_path='',
         # caller must set the logical_file attribute of the context manager
         # before existing context manager
         self.logical_file = None
+        # caller needs to set this to True if aggregation needs to be deleted in the case of an invalid aggregation
+        # created
+        self.remove_logical_file = False
```

Comment:
Instead of creating a new field that needs to be set/checked, could we catch the validationerror that each type raises and remove the logical file on that condition?

Reply:
Made the change.